extensions/amp-analytics/0.1/vendors.js

```diff
@@ -458,6 +459,16 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'adobeanalytics_iframePing': {
+    'triggers': {
+      'pageLoad': {
+        'on': 'visible',
+        'request': 'iframeMessage',
+        'iframePing': true,
```

Comment:
@admoseremic As a suffix. What I'm trying to get at is that we would prefer to not encode any implementation details in the name. PostMessage is not involved here anyway. It is a one-way iframe communication.